test/components/labelTests.ts

```diff
@@ -1,198 +1,240 @@
-    let longLabel = new Plottable.Components.Label("LONG LABELLLLLLLLLLLLLLLLL").xAlignment("left");
-    let topLabel = new Plottable.Components.Label("label").yAlignment("bottom");
-    new Plottable.Components.Table([[topLabel], [testLabel], [longLabel]]).renderTo(svg);
-
-    let testTextRect = (<any> testLabel)._element.select("text").node().getBoundingClientRect();
-    let longTextRect = (<any> longLabel)._element.select("text").node().getBoundingClientRect();
-
-    assert.closeTo(testTextRect.left, longTextRect.left + 30, 2, "left difference by padding amount");
-
-    testLabel.xAlignment("right");
-
-    testTextRect = (<any> testLabel)._element.select("text").node().getBoundingClientRect();
-    longTextRect = (<any> longLabel)._element.select("text").node().getBoundingClientRect();
-
-    assert.closeTo(testTextRect.right, longTextRect.right - 30, 2, "right difference by padding amount");
-
-    testLabel.yAlignment("bottom");
-
-    testTextRect = (<any> testLabel)._element.select("text").node().getBoundingClientRect();
-    longTextRect = (<any> longLabel)._element.select("text").node().getBoundingClientRect();
-
-    assert.closeTo(testTextRect.bottom, longTextRect.top - 30, 2, "vertical difference by padding amount");
-
-    testLabel.yAlignment("top");
-
-    testTextRect = (<any> testLabel)._element.select("text").node().getBoundingClientRect();
-    let topTextRect = (<any> topLabel)._element.select("text").node().getBoundingClientRect();
-
-    assert.closeTo(testTextRect.top, topTextRect.bottom + 30, 2, "vertical difference by padding amount");
-    svg.remove();
-  });
-
-  it("padding puts space around the label", () => {
-    let svg = TestMethods.generateSVG(400, 200);
-    let testLabel = new Plottable.Components.Label("testing label").padding(30);
-    testLabel.renderTo(svg);
-
-    let measurer = new SVGTypewriter.Measurers.Measurer(svg);
-    let measure = measurer.measure("testing label");
-    assert.operator(testLabel.width(), ">", measure.width, "padding increases size of the component");
-    assert.operator(testLabel.width(), "<=", measure.width + 2 * testLabel.padding(), "width at most incorporates full padding amount");
-    assert.operator(testLabel.height(), ">", measure.height, "padding increases size of the component");
-    assert.operator(testLabel.height(), ">=", measure.height + 2 * testLabel.padding(), "height at most incorporates full padding amount");
-    svg.remove();
+  describe("TitleLabel", () => {
+    it("has appropriate css class", () => {
+      const label = new Plottable.Components.TitleLabel("A CHART TITLE");
+      const svg = TestMethods.generateSVG();
+      label.renderTo(svg);
+      const element = (<any> label)._element;
+      assert.isTrue(element.classed(LABEL_CLASS), "element has label css class");
+      assert.isTrue(element.classed(Plottable.Components.TitleLabel.TITLE_LABEL_CLASS), "element has title-label css class");
```

Comment:
Probably sufficient to do `label.classed()`, since the `Component` tests already confirm that that works.